tox.ini

```diff
@@ -2,9 +2,18 @@
 envlist = py36,py37,py38
 
 [testenv]
-extras = test
-commands=
-    python -m pytest --cov titiler --cov-report xml --cov-report term-missing --ignore=venv
+deps =
+    pytest
+    pytest-cov
+    pytest-asyncio
+    requests
+skip_install = true
+commands_pre =
+    python -m pip install titiler/core
+    python -m pip install titiler/application
+commands =
+    python -m pytest titiler/core --cov titiler.core --cov-report xml --cov-report term-missing --ignore=venv
+    python -m pytest titiler/application --cov titiler.application --cov-report xml --cov-report term-missing --ignore=venv
```

Comment:
TODO: cov reports get overwritten

Reply:
Use
```
coverage run --parallel-mode
```
in each module then
```
coverage combine modules/*
```
I don't know how it works through `pytest-cov`; I use `coverage` directly.